docs/advanced-stream.md

```diff
@@ -1,1344 +1,131 @@
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-68-bartholomew-finds-his-way/2797
+* imageUrl: /static/livestream-games/bartholomew.png
 
 ---
 
-* name: Choose Your Own Weapon
-* description: Join Daryl and Vivian as we build a space shooter with lots of different crazy sci-fi weapons!
-* youTubeId: nLE0y9JVm3c
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/nLE0y9JVm3c.jpg
+* name: Alexander's Library
+* description: Alexander is a mouse who is also a librarian. Use his power of shushing to prevent the cats from making a mess of things.
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-45-alexanders-library/2264
+* imageUrl: /static/livestream-games/alexander.png
 
 ---
 
-* name: SUCROMON - Pt.7 Finale!
-* description: Join Richard, Daryl, Shannon, Joey, and Vivian as we finish our seven part series to build "SUCROMON", the monster collecting game where your creatures are sugary delights. In this stream we add XP to our sucromon and handle the enemy and our sucromon running low on health.
-* youTubeId: 7YTq2pdwMek
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/7YTq2pdwMek.jpg
+* name: Giraf-fight
+* description: Giraffes fight to defend their hats. This game is multiplayer, so grab a friend and use both sides of the keyboard!
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-131-giraf-fight-pt-2/3974/3
+* imageUrl: /static/livestream-games/giraf-fight.png
 
 ---
 
-* name: That strawberry is for the feast! Caw?
-* description: Join Richard, Daryl, Shannon, Joey, Vivian, and new guest ERIC as we build a game-idea-generator game where you're a squirrel valiantly defending a strawberry from confused crows.
-* youTubeId: 8Vu83_KpCmc
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/8Vu83_KpCmc.jpg
+* name: Haunted home ownership
+* description: Some ghosts are trying to deal with unfinished business. Others just care about lawn maintenance.
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-46-haunted-home-ownership/2356
+* imageUrl: /static/livestream-games/hhoa.png
 
 ---
 
-* name: Not very scared-crows (charge-attack)
-* description: MakeCode Arcade Advanced - Not very scared-crows
-* youTubeId: ypsWaS1dX18
-* youTubePlaylistId: PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT
-* imageUrl: /static/advanced-stream/ypsWaS1dX18.jpg
+* name: Gacha jobs
+* description: Why does society focus on dogs with jobs? Other animals can have jobs too. Try and collect all the animals with jobs in the gachapon game!
+* cardType: forumUrl
+* url: https://forum.makecode.com/t/arcade-advanced-stream-112-gacha-jobs/3536/4
+* imageUrl: /static/livestream-games/gacha-jobs.png
 
 ---
 
-* name: PlayList
```

Comment:
This one might be fine to leave in as the last card, since it does point to all the videos / they all have forum posts in their descriptions?